src/codegen/doc/mod.rs

```diff
@@ -79,9 +79,9 @@ macro_rules! impl_function_like_type {
     };
 }
 
-impl_to_stripper_type!(Member, Variant);
+impl_to_stripper_type!(Member, Const);
 impl_to_stripper_type!(Enumeration, Enum);
-impl_to_stripper_type!(Bitfield, Type);
+impl_to_stripper_type!(Bitfield, Struct);
```

Comment:
Just to be sure: This change really needed?

Reply:
Yes, they are declared as struct and need to be stored as such, otherwise rustdoc-stripper doesn't "recognize" them.